stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,1781 @@
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func logarithm<U, V>(_ vector: U,
+                                base: LogarithmBase,
+                                result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    
+                    switch base {
+                    case .two:
+                        vvlog2f(dest.baseAddress!,
+                                src.baseAddress!,
+                                &n)
+                    case .ten:
+                        vvlog10f(dest.baseAddress!,
+                                 src.baseAddress!,
+                                 &n)
+                    }
+                }
+            }
+    }
+    
+    /// Calculates either the base two or base ten logarithm of each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter base: Specifies the logarithm base.
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func logarithm<U, V>(_ vector: U,
+                                base: LogarithmBase,
+                                result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    
+                    switch base {
+                    case .two:
+                        vvlog2(dest.baseAddress!,
+                               src.baseAddress!,
+                               &n)
+                    case .ten:
+                        vvlog10(dest.baseAddress!,
+                                src.baseAddress!,
+                                &n)
+                    }
+                }
+            }
+    }
+    
+    // MARK: Unbiased Exponent
+    
+    /// Calculates the unbiased exponent of each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func unbiasedExponent<U, V>(_ vector: U,
```

Comment:
This should just be `exponent`, and the comment should explain that the value is `floor(log2(vector))`.

Reply:
fixed.